benchmark/test_reduction_perf.py

```diff
@@ -286,3 +286,69 @@ def test_perf_vector_norm():
         sizes=SIZES,
     )
     bench.run()
+
+
+def test_perf_slice_scatter():
+    def slice_scatter_args(dtype, batch, size):
+        shape = [batch, size]
+        inp = torch.randn(shape, dtype=dtype, device="cuda")
+        import random
+
+        dim = random.choice([0, 1])
+        start = random.choice([16, 32, 64])
+        end = random.choice([32, 64, 128, 256])
+        step = random.choice([1, 3, 6])
```

Comment:
choosing one of them is enough for benchmark

Reply:
done